### PR TITLE
Submerge toggle button

### PR DESCRIPTION
**Pull request by reinscha**
Added button for submerge ship. Only shows when it should

### CODE REVIEW

**Comment by MannyB6** [comment]
This may not be garbage.

File: src/main/java/assets/game.js [hunk 1]
```diff
@@ -167,6 +170,17 @@ function showMoveFleet() {
...
 }
 
+// Arin's hide the submerge options
+function hideSubmeregeButton(){
+    document.getElementById('subcheck').style.display = "none";
+    document.getElementById('subword').style.display = "none";
+}
+
+// Arin's show the submerge  options when the sub is selected
+function showSubmeregeButton(){
```
Comment (MannyB6) [inline src/main/java/assets/game.js]: Nice job in getting the buttons to hide from the beginning the game is launched.

File: src/main/java/assets/css/custom.css [hunk 2]
```diff
@@ -321,60 +388,17 @@ header{
 
     }
 
-/* Carson Adding buttons */
+/*  Needs Adjusting */
```
Comment (PembleC) [inline src/main/java/assets/css/custom.css]: oh I see how it is ;)

File: src/main/java/assets/game.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ var placedShips = 0;
 var game;
 var shipType;
 var vertical;
+var submerge;
```
Comment (PembleC) [inline src/main/java/assets/game.js]: Still a good idea

File: src/main/java/assets/game.js [hunk 4]
```diff
@@ -167,6 +170,17 @@ function showMoveFleet() {
     document.getElementById("moveFleetSouthButton").style.display = "block";
 }
 
+// Arin's hide the submerge options
```
Comment (PembleC) [inline src/main/java/assets/game.js]: looks familiar

File: src/main/java/assets/game.js [hunk 5]
```diff
@@ -421,19 +434,23 @@ function initGame() {
...
     document.getElementById("place_battleship").addEventListener("click", function(e) {
         shipType = "BATTLESHIP";
        registerCellListener(place(4));
+       hideSubmeregeButton();
     });
     // WE NEED TO FIGURE THIS PART OUT - Carson
     document.getElementById("place_submarine").addEventListener("click", function(e) {
             shipType = "SUBMARINE";
            registerCellListener(place(5));
+          showSubmeregeButton();
```
Comment (PembleC) [inline src/main/java/assets/game.js]: And that is the key! Nice touch!

File: src/main/java/views/ApplicationController/index.ftl.html [hunk 6]
```diff
@@ -87,49 +87,50 @@ <H3>Player Ships Sunk</H3>
...
-            </tr>
-            <tr>
-                <td>
-                    <div class="checkboxVertical">
-                        <input type="checkbox" value="1" id="is_vertical" name=""/>
-                        <label for="is_vertical"></label>
-                    </div>
-                </td>
-            </tr>
+        <tr align="center">
```
Comment (PembleC) [inline src/main/java/views/ApplicationController/index.ftl.html]: Why this?

**Comment by PembleC** [comment]
If it looks good and Manny approves, I saw we merge it!

File: src/main/java/assets/game.js (hunk 4 above)
Comment (reinscha) [reply src/main/java/assets/game.js]: Nope did just copy your function ;)

**Pull request merged**